src/services/schemas/SchemasService.ts

```diff
@@ -25,7 +25,7 @@ export class SchemaService {
     let deferred: ng.IDeferred<ISchemaDefinition> = this.$q.defer<ISchemaDefinition>();
     this.findTarget(container.serverGroup)
       .then(coordinator => {
-          return this.dmrService.executePost({
```

Comment:
We need this return, as this means that the result of the promise `this.dmrService.executePost(...` is passed to L35's then clause. It does not mean that the overarching function `getProtoSchema` returns this result.